src/modules/jcms_admin/jcms_admin.info.yml

```diff
@@ -3,6 +3,5 @@ type: module
 description: Admin improvements.
 core: 8.x
 dependencies:
-  - content_moderation
```

Comment:
Removing this dependency allows me to uninstall the content_moderation module in an update hook. The module is quickly reinstalled when the `../vendor/bin/drush config-import --yes` command is run. As long as we deploy this when the covers preview queue is in the same state as the covers production queue then this should have not noticeable impact.